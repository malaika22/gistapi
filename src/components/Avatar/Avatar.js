import styled from 'styled-components';
import PropTypes from 'prop-types';
export const Avatar = ({ image }) => {
  return <AvatarWrapper src={image} />;
};

const AvatarWrapper = styled.img.attrs((props) => ({
  src: props.src
}))`
  width: 50px;
  height: 50px;
  border-radius: 50%;
`;

Avatar.propTypes = {
  image: PropTypes.string.isRequired
};
