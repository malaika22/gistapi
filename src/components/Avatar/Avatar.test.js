import { render } from '@testing-library/react';
import { Avatar } from './Avatar';

describe('Avatar', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<Avatar />);
    expect(baseElement).toBeTruthy();
  });
  it('should match snapshot', () => {
    const { asFragment } = render(<Avatar />);
    expect(asFragment()).toMatchSnapshot();
  });
});
