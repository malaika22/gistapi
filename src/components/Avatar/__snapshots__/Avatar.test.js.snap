// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar should match snapshot 1`] = `
<DocumentFragment>
  <img
    class="sc-bgqQcB kwIcXm"
  />
</DocumentFragment>
`;

exports[`MedzBreadcrumb should match snapshot 1`] = `
<DocumentFragment>
  <img
    class="sc-bgqQcB kwIcXm"
  />
</DocumentFragment>
`;
